packages/drivers/routerlicious-driver/src/test/wholeSummaryDocumentStorageService.spec.ts

```diff
@@ -0,0 +1,149 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/* eslint-disable max-len */
+
+import assert from "assert";
+import { DebugLogger } from "@fluidframework/telemetry-utils";
+import {
+    SummaryType,
+} from "@fluidframework/protocol-definitions";
+import {
+    IWholeFlatSummary,
+    IWholeFlatSummaryBlob,
+    IWholeFlatSummaryTreeEntry,
+} from "@fluidframework/server-services-client";
+import { WholeSummaryDocumentStorageService } from "../wholeSummaryDocumentStorageService";
+
+const summaryBlobs: IWholeFlatSummaryBlob[] = [
```

Comment:
Can you please add a comment what these are used for.

Reply:
Basically, you are building a snapshot to be returned by downloadSummary.

Reply:
Added more comments throughout test file here.